movieApp/index.html

```diff
@@ -3,25 +3,33 @@
     <head>
     <title>MovieApp</title>
     <meta charset="UTF-8">
-    <link rel="stylesheet" href="/movieApp/styles.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
     </head>
     <body>
-
-        
-    <header id="banner" class="body">
-    <h1><a href="#">Movie App </a></h1>
-  
-    <nav><ul class="topNavBar">
-      <li><a href="#">HOME</a></li>
-      <li><a href="#">GENRES</a></li>
-  
-      <li><a href="#">KIDS</a></li>
-      
-    </ul></nav>
-  
-  
-        
-        <footer>footer</footer>
-        
-    </body>
-</html>
\ No newline at end of file
+    <header>
+      <nav>
+        <label class="logo">MFA</label>
+        <label class="logoText">Movie Finder App</label>   
+        <ul>
+          <li><a href="#">HOME</a></li>
+          <li><a href="#">GENRES</a>
+            <!--  <ul class="subMenu">
```

Comment:
Suggestion for future: don't leave commented code in something which you are going to put for review and then merge to master. 
For now it's fine, just a hint :)